rtv/config.py

```diff
@@ -67,6 +67,9 @@ def build_parser():
     parser.add_argument(
         '--enable-media', dest='enable_media', action='store_const', const=True,
         help='Open external links using programs defined in the mailcap config')
+    parser.add_argument(
+        '--max-comment-cols', dest='max_comment_cols', type=int,
+        help='Maximum comment column width')
```

Comment:
I don't think that a command line flag is really necessary for this setting because it's obscure enough that most people won't be messing with it. I think that having it in the config file is good enough.